TheBTeam.BLL/Services/Edits.cs

```diff
@@ -0,0 +1,297 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TheBTeam.BLL.Model;
+using TheBTeam.BLL.Validators;
+
+namespace TheBTeam.BLL.Services
+{
+    public class Edits
+    {
+        const int MinAddressLength = 3;
```

Comment:
guess it can be took from consolefactory

Reply:
oh yeah its super duper copy pasted, but it feels like a bad practie to mix the namespaces, this will keep evolving once we switch to the web version anyway, the "anyway" way of thinking feels like a bad practice too tho ¯\_(ツ)_/¯

Reply:
Fine, but Will you remember to change it everywhere?